ChangeLog.rst

```diff
@@ -9,6 +9,10 @@ Breaking changes
 be lost. Please see the pull-request description for manual steps required after
 upgrading a MetalK8s 0.1 cluster to MetalK8s 0.2 (:ghissue:`147`)
 
+:ghpull:`94` - flatten the storage configuration and allow more user defined
+storage related actions. please check the chapter
+"Upgrading from MetalK8s < 0.2.0" (:ghissue:`153`)
```

Comment:
Let's turn this into a real reference, see http://www.sphinx-doc.org/en/stable/markup/inline.html#role-ref